packages/react-table/src/components/TableComposable/examples/ComposableTable.md

```diff
@@ -340,6 +340,13 @@ To make multiple columns sticky, wrap `TableComposable` with `InnerScrollContain
 ```ts file="ComposableTableMultipleStickyColumns.tsx"
 ```
 
+### Composable: Multiple right-aligned sticky columns
+
+To make multiple columns sticky, wrap `TableComposable` with `InnerScrollContainer` and add `isStickyColumn` to all columns that should be sticky. The leftmost sticky column should also have the `hasLeftBorder` property, and each sticky column left of the last column must define a `stickyRightOffset` property that equals the combined width of the previous sticky columns - set by `stickyMinWidth`. To prevent the default text wrapping behavior and allow horizontal scrolling, all `Th` or `Td` cells should also have the `modifier="nowrap"` property.
```

Comment:
Similar to the prop description updates + a slight tweak the this example description:

```suggestion
To make multiple right-aligned columns sticky: 

- wrap `TableComposable` with `InnerScrollContainer` 
- add `isStickyColumn` to all columns that should be sticky
- add `hasLeftBorder` to the leftmost sticky column 
- pass in `stickyRightOffset` to each sticky column preceding the last column, with a value that equals the combined width - set by `stickyMindWidth` - of each sticky column that comes after the current one

To prevent the default text wrapping behavior and allow horizontal scrolling, all `Th` or `Td` cells should also have the `modifier="nowrap"` property.
```

The verbiage for right aligned sticky feels a little trickier to get right, so let me know what you think.

Depending what you think of using a list here, the "Composable: Multiple sticky columns" example description could also be updated (maybe also update the example name to "Composable: Multiple left-aligned sticky columns"?)